src/plugins/dashboard/public/dashboard_container/embeddable/dashboard_container.tsx

```diff
@@ -134,7 +136,9 @@ export class DashboardContainer
     TrackContentfulRender,
     HasSaveNotification,
     HasRuntimeChildState,
-    HasSerializedChildState
+    HasSerializedChildState,
+    PublishesSettings,
+    Partial<PublishesViewMode>
```

Comment:
These were removed from the `PresentationContainer`, since the control group (which is **also** a container) does not do these things - these are specific to dashboard, not all containers.